### PR TITLE
docs: add section on importing the Grafana dashboard using the Grafana Helm Chart

### PR DESCRIPTION
**Pull request by maritiren**
## Description
Adding section on how to import the dashboard using the Grafana Helm Chart. I think this is helpful as many use the Grafana Helm chart to set it up. 

From the Grafana spec:
https://github.com/grafana/helm-charts/blob/grafana-8.0.2/charts/grafana/values.yaml#L708-L759

## Checklist
- [x] I've read the [guidelines for contributing](https://github.com/aquasecurity/trivy-operator/blob/main/CONTRIBUTING.md) to this repository.
- [ ] I've added tests that prove my fix is effective or that my feature works. 
- [x] I've updated the [documentation](https://github.com/aquasecurity/trivy-operator/tree/main/docs) with the relevant information (if needed).
- [ ] I've added usage information (if the PR introduces new options)
- [ ] I've included a "before" and "after" example to the description (if the PR is a user interface change).

### CODE REVIEW

File: docs/tutorials/grafana-dashboard.md [hunk 1]
```diff
@@ -200,8 +205,36 @@ Once you see all the default Dashboards, click `New`, then `Import`.
...
+          path: /var/lib/grafana/dashboards/default
+  dashboards:
+    default: 
+      trivy-operator-dashboard:
+        gnetId: 17813
+        revision: 2
+        datasource: Prometheus
+```
+
+When the Helm Chart has been applied, you should see the dashboard as part of your Dashboard list. It is named `Trivy Operator Dashboard`.
```
Comment (maritiren) [inline docs/tutorials/grafana-dashboard.md:238]:
```suggestion
When the Helm chart has been applied, you should see the dashboard as part of your Dashboard list. It is named `Trivy Operator Dashboard`.
```

File: docs/tutorials/grafana-dashboard.md [hunk 2]
```diff
@@ -186,8 +186,13 @@ sum(trivy_image_exposedsecrets)
 ### Set up Grafana Dashboard for Trivy Operator Metrics
 
 Lastly, we want to visualise the security issues within our cluster in a Grafana Dashboard.
+You can either do that manually in Grafana, or through the Helm Chart.
```
Comment (maritiren) [inline docs/tutorials/grafana-dashboard.md:189]:
```suggestion
You can either do that manually in Grafana, or using the Helm chart. 
```

File: docs/tutorials/grafana-dashboard.md [hunk 3]
```diff
@@ -200,8 +205,36 @@ Once you see all the default Dashboards, click `New`, then `Import`.
 
 Here, we will paste the ID of the Aqua Trivy Dashboard: `17813`
 
-The link to the dashboard in Grafana is [the following.](https://grafana.com/grafana/dashboards/17813)
-
-Once pasted, you should see the following Dashboard as part of your Dashboard list: `Trivy Operator Dashboard`
+Once pasted, you should see the following dashboard as part of your Dashboard list: `Trivy Operator Dashboard`
+
+
+#### Using the Helm Chart
```
Comment (maritiren) [inline docs/tutorials/grafana-dashboard.md:211]:
```suggestion
#### Using the Grafana Helm Chart
```

File: docs/tutorials/grafana-dashboard.md [hunk 4]
```diff
@@ -186,8 +186,13 @@ sum(trivy_image_exposedsecrets)
 ### Set up Grafana Dashboard for Trivy Operator Metrics
 
 Lastly, we want to visualise the security issues within our cluster in a Grafana Dashboard.
+You can either do that manually in Grafana, or using the Helm chart.
```
Comment (maritiren) [inline docs/tutorials/grafana-dashboard.md:189]:
```suggestion
You can either do that manually in Grafana, or using the Grafana Helm chart. 
```

**Review by chen-keinan — APPROVED** [review]
@maritiren lgtm 🚀

**Pull request merged**